### PR TITLE
Adding Tennis Academy Dash

### PR DESCRIPTION
**Pull request by iolivia**
Adding my game to this month's update 😄 

-----

_Part of #64 ("Newsletter 7: Coordination/Tracking")_

### CODE REVIEW

**Comment by 17cupsofcoffee** [comment]
(Complete aside from the actual PR: this game has come a massive way since I last saw it - looks really cool 😄)

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -90,6 +90,16 @@ This month, the focus was on the cooking skill. This includes recipies and cookw
 [antorum]: https://dooskington.com
 [@dooskington]: https://twitter.com/dooskington
 
+### [Tennis Academy Dash][tennis-academy-dash]
+
+![release](dash.gif)
+
+Tennis academy dash is a time management game where you are the manager of a tennis academy and you need to coordinate various players to play on your courts. 
+
+The game is still a work in progress, but the demo version has been uploaded to itch.io this month, so go ahead and check it out! Give it a go (the build is only for mac at the moment but other platforms coming soon) and drop a comment with any feedback on [tennis-academy-dash].
```
Comment (AlexEne) [inline content/posts/newsletter-007/index.md]: ` so go ahead and check it out!` -> I would change this into something less imperative, as in `so you can go ahead and...`

Comment (ozkriff) [reply content/posts/newsletter-007/index.md]: :+1: I'll tweak this in the final draft.

**Comment by ozkriff** [comment]
Thanks for the PR!

**Comment by iolivia** [comment]
thanks everyone! btw I really like the open newsletter style, I think it's really encouraging others to participate 😄

**Pull request merged**